app/src/main/java/co/devhack/compose/southamericanqualifiers/ui/activities/DetailTeamActivity.kt

```diff
@@ -0,0 +1,136 @@
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.text.BasicText
+import androidx.compose.material.Card
+import androidx.compose.material.Divider
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.setContent
+import androidx.compose.ui.res.loadVectorResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.ui.tooling.preview.Preview
+import co.devhack.compose.southamericanqualifiers.R
+import co.devhack.compose.southamericanqualifiers.ui.commons.SouthAmericanQualifiersTheme
+
+class DetailTeamActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            Content()
+        }
+    }
+}
+
+@Composable
+fun Content() {
+    SouthAmericanQualifiersTheme {
+        Surface(color = MaterialTheme.colors.background) {
+            ContentDetailTeam(
+                nameTeam = "Argentina",
+                flagTeam = R.drawable.ic_launcher_background,
+                matchesWon = 5,
+                matchesDraw = 5,
+                matchesLost = 3
+            )
+        }
+    }
+}
+
+@Composable
+fun ContentDetailTeam(
+    nameTeam: String,
+    flagTeam: Int,
+    matchesWon: Int,
+    matchesLost: Int,
+    matchesDraw: Int
+) {
+    Column {
+        DetailHeadTeam(nameTeam, flagTeam)
+        Divider(color = Color.Blue)
+        DetailMatchTeam(matchesWon, matchesLost, matchesDraw)
+    }
+}
+
+@Composable
+fun DetailHeadTeam(nameTeam: String, flagTeam: Int) {
+    Card(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(8.dp)
+            .height(100.dp),
+        elevation = 8.dp,
+        border = BorderStroke(2.dp, Color.Blue)
+    ) {
+        Row {
+            BasicText(nameTeam)
+            loadVectorResource(id = flagTeam).resource.resource?.let {
+                Image(it)
+            }
+        }
+    }
+}
+
+@Composable
+fun DetailMatchTeam(matchesWon: Int, matchesLost: Int, matchesDraw: Int) {
+    Card(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(8.dp)
+            .height(50.dp),
+        elevation = 8.dp,
+        border = BorderStroke(2.dp, Color.Blue)
+    ) {
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            Column(
+                modifier = Modifier
+                    .padding(start = 8.dp)
+                    .align(Alignment.CenterVertically)
+            ) {
+                BasicText("PG", style = TextStyle(fontWeight = FontWeight.Bold))
```

Comment:
Same here, `Text` instead of `BasicText`.

Also, we should be creating the string in the `strings.xml` file, and then referencing it with `stringResource(R.string.text_id)`